cpp/arcticdb/version/local_versioned_engine.cpp

```diff
@@ -986,7 +986,8 @@ std::vector<timestamp> LocalVersionedEngine::batch_get_update_times(
     return results;
 }
 
-std::shared_ptr<std::unordered_map<std::pair<StreamId, VersionId>, AtomKey>> LocalVersionedEngine::get_stream_index_map(
+std::pair<std::shared_ptr<std::unordered_map<std::pair<StreamId, VersionId>, AtomKey>>, 
+std::shared_ptr<std::unordered_map<StreamId, AtomKey>>> LocalVersionedEngine::get_stream_index_map(
```

Comment:
We don't have an official style guide yet, but please make such long return type more readable. For example, consider indenting like:

```cpp
std::pair<std::shared_ptr<std::unordered_map<std::pair<StreamId, VersionId>, AtomKey>>, 
          std::shared_ptr<std::unordered_map<StreamId, AtomKey>>>
LocalVersionedEngine::get_stream_index_map(
```

Reply:
done